content/tutorial/03-umbrella-sdk/01-unified-approach/02-complete-workflow/README.md

```
@@ -0,0 +1,112 @@
+Now that we understand the CowSdk basics, let's build a complete trading workflow using the unified SDK. We'll create an order in three steps:
+
+1. Generate app data (metadata)
+2. Get a quote (pricing)
+3. Sign the order (authorization)
+
+## Step 1: Generate App Data
+
+```typescript
+/// file: run.ts
+const appDataDoc = await cowSdk.metadataApi.generateAppDataDoc({
+	appCode: 'Decentralized CoW',
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)